airflow/www/static/js/connection_form.js

```diff
@@ -55,6 +55,10 @@ $(document).ready(function () {
         'host': 'https://<env>.qubole.com/api'
       }
     },
+    kubernetes: {
+      hidden_fields: ['host', 'schema', 'login', 'password', 'port', 'extra'],
```

Comment:
Why are we hiding host? I would expect this to be where I would configure the kube api server when not running in a cluster.

Reply:
when not running in a cluster i want to get kube config in JSON format,
Kube config is the Kubernetes connection string.
https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/